host/4/bullseye/amd64/powershell/powershell7-core-tools.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:6.0.100
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1
```

Comment:
V4 images for powershell 7 should include both Net Core 3.1 SDK and .Net 6.0 SDK?

Reply:
It depends if this container needs to support PowerShell 7.2 as well. I think that needs .NET 6. Otherwise just .NET Core 3.1 should be fine for PowerShell 7.

Reply:
My recommendation - For consistency we should include .Net 6 SDK in V4 images - as this is part of all other images.

Reply:
Yeah actually having .NET 6 SDK would be good for supporting manually installed extensions with extensions.csproj.

Reply:
Added dotnet6 sdk to the image :) 

```root@02261c5fae2b:/# dotnet --list-runtimes
Microsoft.AspNetCore.App 3.1.22 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
Microsoft.AspNetCore.App 6.0.0 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
Microsoft.NETCore.App 3.1.22 [/usr/share/dotnet/shared/Microsoft.NETCore.App]
Microsoft.NETCore.App 6.0.0 [/usr/share/dotnet/shared/Microsoft.NETCore.App]